### ISSUE TITLE
2.5.8: target offset can be determined only with complicated mathematical methods

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2974]
[definition of target offset](https://www.w3.org/TR/WCAG22/#dfn-target-offsets):

> length of the longest possible line that starts at an edge of a target (A), intersects a second edge of A, and ends at the closest edge of a second target (B). 

Even for simple click targets, such as circle and triangle, target offset is not easy to determine. In the figure, a circle and a triangle are positioned horizontally. The triangle is positioned on its baseline.
We are now looking for target offset from the circle to the triangle. First guess would be to take the farthest point from the circle (i.e. in the center). Second guess would be to take the topmost point from the circle, because then the triangle is far away.
Both assumptions are wrong: the farthest point is between the middle and the top point on the circle. How exactly is this point determined? For mathematicians certainly no problem, but for normal people already difficult. If the shapes become more irregular (e.g. image of a map), then also mathematicians cannot apply mathematical calculation methods, but must measure every possible line and then compare all possible lines. 

![258c](https://user-images.githubusercontent.com/52748763/215020684-70053429-3393-47f6-9353-231be7bcbd93.jpg)

Also, I doubt that most people understand what is meant by the definition because it is not explained in more detail in Understanding either. The definition is difficult to understand because a longest line to a closest point is sought. This seems like a contradiction in itself. Mathemtically, finding such a line is possible, but this is a mathematical concept that is certainly not familiar to non-mathematicians. For common sense, there is only the largest and the smallest distance, defined as the distance between the two most distant and the two closest points, respectively

**Comment by alastc** [comment]
Similar to the last issue, from the previous decisions, the group was trying to make common cases (which triangles are not) easier to measure, rather than be the 'correct' metric for all cases.

Circles and triangles are not common target shapes, so optimising the understandability of the metric for those targets (at the expense of typical rectangles) does not make sense.

We do need to update the understanding document though.
